### PR TITLE
data:singhmohit14072002

### PR DESCRIPTION
**Pull request by singhmohit14072002**

### CODE REVIEW

**Comment by singhmohit14072002** [comment]
Hi, I am Contribute first time if it does not have any issues or conflicts can you approve it

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Please change the file name to you github username

**Review by singhmohit14072002 — COMMENTED** [review]
change that

**Review by singhmohit14072002 — COMMENTED** [review]
still, it can't get merge

**Review by singhmohit14072002 — COMMENTED** [review]
I can the file name but still it  does not merge 
can you suggest what happen

**Review by singhmohit14072002 — COMMENTED** [review]
change it

**Review by singhmohit14072002 — COMMENTED** [review]
changed

**Comment by ChinmayMhatre** [comment]
> I change the file name but still it does not merge can you suggest what happen

The PR will be reviewed and merged by the maintainers.

File: data/singhmohit14072002.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Mohit Singh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "OpenSource  | DevOps | Docker | Blogger **Hashnode**",
+  "avatar": "https://github.com/singhmohit14072002.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/https://twitter.com/MohitDevops",
```
Comment (ChinmayMhatre) [inline data/singhmohit14072002.json:10]:
```suggestion
      "url": "https://twitter.com/MohitDevops",
```
Please update the url

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Review by ChinmayMhatre — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge. Great collaboration Everyone! 🎉

<p align="center"></p>

**Pull request merged**